src/kolibri.yaml

```diff
@@ -0,0 +1,35 @@
+---
+description: >
+  The offline app for universal education.
```

Comment:
This part looks more like an ad than a description to me ^^

_Contribute now to Kolibri! **The** offline app for universal education._

I'd feel more comfortable if we'd use `An offline app` instead of `The offline app`